.circleci/check_ineffassign.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# ignore protobuf-generated code, as well as yacc (.y) parser files
+# duplicate output to stderr so we can see it in console while `wc` is counting the lines
+result=$($GOPATH/bin/ineffassign . | grep -v "\.pb\.go\|\.y" | tee /dev/stderr | wc -l)
+
+if [[ $result -gt 10 ]]; then
+  echo "Too many ineffassign issues: $result"
+  exit 1;
```

Comment:
The semicolon `;` is unnecessary here and below. Usually it should not be placed when we use code formatting with line breaks.